File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroCerco.java
package com.rinconadadelsur.sistemas.bioseguridad.Entidades;

public class eRegistroCerco {

    private String _codigo;
    private String _componente;
    private String _descripcion;

    public eRegistroCerco(){

    }

    public eRegistroCerco(String _codigo, String _componente, String _descripcion) {
        this._codigo = _codigo;
        this._componente = _componente;
        this._descripcion = _descripcion;
    }

    public String get_codigo() {
        return _codigo;
    }

    public void set_codigo(String _codigo) {
        this._codigo = _codigo;
    }

    public String get_componente() {
        return _componente;
    }

    public void set_componente(String _componente) {
        this._componente = _componente;
    }

    public String get_descripcion() {
        return _descripcion;
    }

    public void set_descripcion(String _descripcion) {
        this._descripcion = _descripcion;
    }

    @Override
    public String toString() {
        return _descripcion;
    }
}

File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroCerco.kt
package com.rinconadadelsur.sistemas.bioseguridad.Entidades

class eRegistroCerco {
    private var _codigo: String? = null
    private var _componente: String? = null
    private var _descripcion: String? = null

    constructor()

    constructor(_codigo: String?, _componente: String?, _descripcion: String) {
        this._codigo = _codigo
        this._componente = _componente
        this._descripcion = _descripcion
    }

    fun get_codigo(): String? {
        return _codigo
    }

    fun set_codigo(_codigo: String?) {
        this._codigo = _codigo
    }

    fun get_componente(): String? {
        return _componente
    }

    fun set_componente(_componente: String?) {
        this._componente = _componente
    }

    fun get_descripcion(): String {
        return _descripcion!!
    }

    fun set_descripcion(_descripcion: String) {
        this._descripcion = _descripcion
    }

    override fun toString(): String {
        return _descripcion!!
    }
}
